bin/tests/cli.rs

```diff
@@ -1079,8 +1079,9 @@ async fn test_lookback_restarting_agent() {
 }
 
 #[tokio::test]
-#[cfg_attr(not(feature = "integration_tests"), ignore)]
-#[cfg_attr(not(target_os = "linux"), ignore)]
+#[ignore]
+//#[cfg_attr(not(feature = "integration_tests"), ignore)]
```

Comment:
Is commenting out these conditions intentional?

Reply:
If I leave them uncommented clippy complains as the blanket ignore means they're never hit. I'm working on a fix for the processing of symlinks on init now